clcache.py

```diff
@@ -1482,9 +1520,11 @@ def processCompileRequest(cache, compiler, args):
         else:
             assert objectFile is not None
             if 'CLCACHE_NODIRECT' in os.environ:
-                return processNoDirect(cache, objectFile, compiler, cmdLine, environment)
+                compilerResult = processNoDirect(cache, objectFile, compiler, cmdLine, environment)
             else:
-                return processDirect(cache, objectFile, compiler, cmdLine, sourceFiles[0])
+                compilerResult = processDirect(cache, objectFile, compiler, cmdLine, sourceFiles[0])
+            printTraceStatement("Finished. Exit code {0:d}".format(compilerResult[0]))
+            return compilerResult
```

Comment:
In principle I could, but IIRC I did it because I proceeded to do big changes to `processDirect` and `processNoDirect` which were simplified by not having that print statement. I'll see whether I can tear it out (and then fixup the subsequent commits) without too much work.